spring-integration-mqtt/src/main/resources/org/springframework/integration/mqtt/config/spring-integration-mqtt.xsd

```diff
@@ -89,6 +89,14 @@
 					</xsd:documentation>
 				</xsd:annotation>
 			</xsd:attribute>
+			<xsd:attribute name="manual-acks" type="xsd:boolean" default="false">
```

Comment:
In other places we do like this:
```
<xsd:simpleType>
	<xsd:union memberTypes="xsd:boolean xsd:string"/>
</xsd:simpleType>
```
to avoid validation errors in the IDE when we resolve a value against properties placeholder.
Search for some `boolean` exactly in this `spring-integration-mqtt.xsd`

Reply:
I thought about this, but it is unlikely that someone would set this via a property - the code either manually acks or not.

Reply:
But I can change it if you insist.

Reply:
No, I'm OK with your explanation.
More over it doesn't look like people worry about XML config these days too much 😉